lib/vectors.py

```diff
@@ -80,6 +80,14 @@ def __eq__(self, other_vec):
     def __str__(self):
         return str((self.x, self.y, self.z))
 
+    def distance(self, other):
+        diff = other - self
```

Comment:
Nota that `diff` is no longer used.

Reply:
If people ever complain about how slow the program is, we can then remove this statement, and they can marvel at the speedup.